### DISCUSSION TITLE
Renovate keepUpdatedLabel clarification

### DISCUSSION THREAD

**Discussion by bholland-bh** [renovatebot/renovate#32609] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github
`ghcr.io/renovatebot/renovate:38.142.7-full`

### Please tell us more about your question or problem

I have enabled the `keepUpdatedLabel` in my renovate configuration.
```yaml
- name: RENOVATE_KEEP_UPDATED_LABEL
  value: "keep-updated"
```

I have added the label to a PR that I created (not renovate), and renovate is not rebasing this PR. I checked the logs, and my branch is not even detected.

Is that because I was the one who created the PR?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Renovate ignores PRs which aren't its own.
